compose-jetsnack/android/build.gradle.kts

```diff
@@ -37,6 +37,6 @@ android {
 }
 
 compose {
-   val composeVersion = project.property("compose.version") as String
+   val composeVersion = project.property("compose.compiler.version") as String
```

Comment:
Is it still required?

Reply:
Hm. I thought I removed all instances of this.
I'll remove it